docs/development/getting-started-locally.md

```diff
@@ -0,0 +1,137 @@
+# Etcd-Druid Local Setup
+
+This page aims to provide steps on how to setup Etcd-Druid locally with/without storage providers
+
+## Prerequisites 
```

Comment:
Don't we need some more prerequisites? 
maybe kind, since later on in this doc there is a section on starting a kind cluster?

Reply:
We need some prerequisites such as kind and helm but they will be auto installed if not present already when we trigger `make kind-up` & `make deploy` respectively. So I removed them from prequisities.